framework/src/engine/consensus/certificate_generation/commit_pool.ts

```diff
@@ -330,26 +342,39 @@ export class CommitPool {
 				...this._nonGossipedCommitsLocal.getByHeight(nextHeight),
 				...this._gossipedCommits.getByHeight(nextHeight),
 			];
-			const nextValidators = singleCommits.map(commit => commit.validatorAddress);
 			let aggregateBFTWeight = BigInt(0);
 
 			// Assume BFT parameters exist for next height
 			const { validators: bftParamValidators, certificateThreshold } =
 				await this._bftMethod.getBFTParameters(methodContext, nextHeight);
```

Comment:
We can replace this by `getBFTParametersActiveValidators`

Reply:
if we replace it with `getBFTParametersActiveValidators` the problem is that it will throw error `Validator address not found in commit pool` bc there are commits from the standby validators which will not be in `getBFTParametersActiveValidators`